client/src/js/components/bhFiscalPeriodSelect.js

```diff
@@ -1,6 +1,9 @@
 angular.module('bhima.components')
   .component('bhFiscalPeriodSelect', {
     bindings : {
+      fiscalId : '<',
+      periodFrom : '<',
+      periodTo : '<',
```

Comment:
These are IDs, right?  It might make more sense to call them `periodFromId`, etc, to make that explicit.